third_party/cereal/BUILD

```diff
@@ -0,0 +1,126 @@
+licenses(["notice"])
```

Comment:
We already have a cereal lib in the third_party dir [here](https://github.com/apache/incubator-heron/tree/master/third_party/cereal).

So please either remove the files here or send out another separate PR for changing this lib